config/badgegenerator.yml

```diff
@@ -1 +1,4 @@
-USER_DATA: "{\"default\":{\"project_public_key\":\"aabbcc\",\"private_key\":\"123456\",\"epochs_max_cu\":1},\"projectId2\":{\"project_public_key\":\"aabbccdd\",\"private_key\":\"12345678\",\"epochs_max_cu\":1}}"
\ No newline at end of file
+USER_DATA: "{\"1\":{\"default\":{\"project_public_key\":\"test111111\",\"private_key\":\"123456\",\"epochs_max_cu\":1},\"projectId2\":{\"project_public_key\":\"test1122\",\"private_key\":\"12345678\",\"epochs_max_cu\":1}},\"2\":{\"default\":{\"project_public_key\":\"test123\",\"private_key\":\"123456\",\"epochs_max_cu\":1}}}"
+DEFAULT_GEOLOCATION: 1
+COUNTRIES_FILE_PATH: "~/projects/lavatools/countries.csv"
+IP_FILE_PATH: "~/Downloads/ip2asn-v4.tsv"
```

Comment:
please add a sample file to the `config/badgegenerator` folder and use them in the configuration so we will have during development and testing.